build/ChangeTargetFramework.ps1

```diff
@@ -0,0 +1,31 @@
+﻿Param(
+  [string]$TFparam
+)
+
+$newTF = "netstandard2.0;net45;netstandard1.6"
+
+If (![string]::IsNullOrEmpty($TFparam)) 
+{
+	$newTF = $TFparam
+}
+
+$projectFiles = Get-Childitem .. -Recurse -Include *.csproj
+
+foreach($currentProjFile in $projectFiles)
+{
+  $isTest = $currentProjFile.Name.Contains(".Tests.csproj")  
+
+  # test files are untouched - they are just using only one platform anyway
+  if($isTest) { continue }
```

Comment:
You don't need this if-statement when you exclude the tests .csproj in $projectFiles. See my suggestion above.